framework/base/ErrorHandler.php

```diff
@@ -100,6 +108,10 @@ public function handleException($exception)
             }
             $this->renderException($exception);
             if (!YII_ENV_TEST) {
+                \Yii::getLogger()->flush(true);
```

Comment:
Yes it's affect non HHVM, but it's make sense (I follows [this line](https://github.com/yiisoft/yii2/pull/8976/files#diff-f000680354101249e9f60e71e6278e3bL193) patterns where it's calling flush before exit).
It's done for `if (!YII_ENV_TEST) {` because the exit only called inside that branch.